network.go

```diff
@@ -97,7 +98,7 @@ type ipInfo struct {
 type svcInfo struct {
 	svcMap     map[string][]net.IP
 	svcIPv6Map map[string][]net.IP
-	ipMap      map[string]*ipInfo
+	ipMap      common.SetMatrix
```

Comment:
I dont quite understand the need to have this ipMap structure to container more than 1 ipInfo ?
Hence I dont see a reason to change it to use SetMatrix.